boa/src/syntax/ast/node/statement_list.rs

```diff
@@ -62,3 +64,24 @@ impl fmt::Display for StatementList {
         self.display(f, 0)
     }
 }
+
+#[derive(Clone, Debug, Finalize, PartialEq)]
+pub struct RcStatementList(Rc<StatementList>);
```

Comment:
This does not need to be `pub` it can be `pub(crate)`. Also some documentation would be nice.

Reply:
hmm we cannot export the crate nor we can use it on any of the builtins. i cannot seem to mark it pub(crate) and just export it as pub(crate)

Reply:
> hmm we cannot export the crate nor we can use it on any of the builtins. i cannot seem to mark it pub(crate) and just export it as pub(crate)

My bad, the `Function` enum is pub so its fields have to be pub too. In this case leave it as pub.

Reply:
Oh ok i leaked that type through the Function enum now i fixed it